src/backend/aspen/api/policy.polar

```diff
@@ -9,22 +9,19 @@ resource Group {
 
 resource Sample {
   roles = ["admin", "viewer", "member"];
-  permissions = [ "read", "read_private", "read_public", "sequences", "write"];
+  permissions = [ "read", "read_private", "sequences", "write"];
```

Comment:
the `read_public` permission made no sense, and even confused me writing tests. I was able to remove it once I wrote the tests, and then the tests and verified that worked properly.